_includes/website/header.html

```diff
@@ -1,5 +1,5 @@
 <nav>
-  <div class="logo"><img src="http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/overview/helix-logo-2.svg" class="logo"></div>
+  <div class="logo"><a href="{{site.baseurl}}"><img src="http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/overview/Helix_Logo_%20Final-2.svg" class="logo"></a></div>
```

Comment:
adding a / after baseurl will fix the issue of it not working locally:  <a href="{{site.baseurl}}/">